style.css

```diff
@@ -915,4 +1006,173 @@ section {
     left: 6rem;
     visibility: hidden;
   }
+
+  .popup {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: #fff;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+    z-index: 9999;
+    width: 70%;
+    height: 85%;
+    overflow: hidden;
+  }
```

Comment:
- [ ] Good job so far implementing the changes Requested by the Previous Reviewers :clap: However, your popup is still not matching the design, Please try to match the design according to the given requirement. 
 
- [ ] Please ensure that the font also matches the given design requirement that says `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using
[this design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-27&mode=design&t=yMHEKO8rVrh7PYJm-0)

below is an illustration of the difference between yours  and the given design :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Desktop Design](https://github.com/josman227/portfolio.github.io/assets/43528594/796e4183-1492-492d-a1ee-a2d00a332828)


</td>
      <td>

![Figma Desktop](https://github.com/josman227/portfolio.github.io/assets/43528594/a8ad8ef8-997c-4566-90c6-80817aa86383)


</td>
    </tr>
  </table> 


<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Mobile Design](https://github.com/josman227/portfolio.github.io/assets/43528594/791b1bed-c157-4117-9894-67abb8db54ed)


</td>
      <td>

![Figma Mobile Design](https://github.com/josman227/portfolio.github.io/assets/43528594/9ab11907-0175-4c8b-a0d4-fe5853451456)

</td>
    </tr>
  </table>